admin/index.js

```diff
@@ -69,4 +71,11 @@ function AccountAdmin (options) {
     one: events.one.bind(this, state),
     off: events.off.bind(this, state)
   }
+
+  // sessions.add can use accounts.find to lookup user id by username
+  admin.sessions = {
+    add: sessionsAdd.bind(admin, state)
+  }
```

Comment:
> sessions.add can use accounts.find to lookup user id by username

that shouldn’t be a problem. Instead of using `this.accounts.find`, you can do something like

```js
var accountsFind = require('./lib/accounts/find')
// ...
accounts.find(state, id)
```

Does that make sense? When you look into our code, you will see that we don’t use `this` at all. All the `signOut.bind(this, state)` lines above could as well be `signOut.bind(null, state)` – in fact I think we should do that change to make it less confusing :)

We pass around the `state` variable instead of using `this`. That makes it much simpler to test methods for unit test, we can simply pass in what ever `state` we want to test, we don’t need to run any constructor or anything for unit tests. It’s functional, not object orientated. 

Hope that makes sense? Please let me know if you get stuck or anything, happy to help out